src/RA_Defs.h

```diff
@@ -53,7 +53,7 @@
 #define NODEFERWINDOWPOS
 #define NOMCX  
 
-
+struct IUnknown;
```

Comment:
The error appeared directly in `ObjBase.h` (a Windows header file).
`Error C2760 syntax error: unexpected token 'identifier', expected 'type specifier'`

Research online suggested to forward declare it before including a Windows Header in a  translation unit. It made sense since `IUnknown `was used before it was defined.

Files affected were `RA_Defs.h, RA_Dlg_Login.h, RA_Dlg_RomChecksum.h, and RA_Dlg_RichPresence.h`